gumroad/src/common.scss

```diff
@@ -0,0 +1,13 @@
+/**
+ * #.# Common SCSS
+ *
+ * Can include things like variables and mixins
+ * that are used across the project.
+*/
+
+// Colors.
+$black: rgb(41, 41, 41);
```

Comment:
don't think we'll need this file. can just have one for all of gumroad. also, white is not f4f4f4!